components/browser/menu/src/main/res/layout/mozac_browser_menu_item_simple.xml

```diff
@@ -3,10 +3,13 @@
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <TextView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/simple_text"
     style="@style/Mozac.Browser.Menu.Item.Text"
     android:layout_width="match_parent"
     android:layout_height="@dimen/mozac_browser_menu_item_container_layout_height"
     android:gravity="center_vertical"
```

Comment:
Do we really need  `android:gravity="center_vertical"`? As the lint error described the `android:textAlignment="viewStart"` and  `android:gravity="center_vertical"` are inconsistent alignments

![image](https://user-images.githubusercontent.com/773158/67232805-145ffd80-f410-11e9-8b55-fd0bea0afaef.png)

Reply:
I think we need the center vertical in order to center the text. The alignments are not inconsistent. It is a compatibility issue with `textAlignment` if you target older versions. It warns you that for versions older than 17 the attribute would be ignored and the gravity would be `center_vertical` when you would expect `center_vertical|start`. The latter is a valid gravity combination.

Reply:
Thanks for the clarification 👍